deliveries/Webb_Mixer_Extended_milestone_1.md

```diff
@@ -0,0 +1,24 @@
+# Milestone Delivery :mailbox:
+
+Payment in DOTS to: 16aN6hVU7WtMD7pw2kjXNwnApQB9tNxcdsknTSyFDps6YAiA
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Open-Grants-Program/pull/422
+* **Milestone Number:** #1
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes. Once the node is up, it will be possible to send test transactions that will show how the new functionality works. |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | https://www.notion.so/hicommonwealth/Blog-post-268628ee32ac4f21b9b8531385b00458 | 
+| 1. | Substrate module: Multi-Asset support for mixer | https://github.com/webb-tools/anon/tree/master/pallets |
+| 1a. | Substrate module: ORML support | https://github.com/webb-tools/anon/tree/master/pallets/mixer |
+| 1b. | Substrate module: Pallet asset support | https://github.com/webb-tools/anon/tree/master/pallets/assets-mixer |
+| 2. | Substrate module: New ZKP backend support | https://github.com/webb-tools/anon/blob/master/pallets/merkle/src/utils/setup.rs |
+| 2a. | Substrate module: New ZKP backend support | https://github.com/webb-tools/anon/blob/master/pallets/merkle/src/utils/setup.rs |
+| 2b. | Substrate module: ZKP backend support | https://github.com/webb-tools/anon/blob/master/pallets/merkle/src/utils/setup.rs |
+| 2c. | Substrate module: New ZKP hash function support | https://github.com/webb-tools/anon/blob/master/pallets/merkle/src/utils/setup.rs |
+| 3. | Zero-knowledge circuits | https://github.com/webb-tools/arkworks-gadgets |  
```

Comment:
Do I understand correctly that this is Apache 2.0, and you also include the MIT one since you're using MIT-licensed code? Could you state this more explicitly or rename the first file to just `LICENSE`?

Reply:
Yep, all done.